doc/usage.rst

```diff
@@ -39,6 +39,30 @@ of ``target_link_libraries`` with a customized version::
   include(CeleritasLibrary)
   celeritas_target_link_libraries(mycode PUBLIC Celeritas::celeritas)
 
+As ``celeritas_target_link_libraries`` decays to ``target_link_libraries`` if
+CUDA and VecGeom are not used, you can use it safely to link nearly all targets consuming
+Celeritas, or Celeritas consumers, in your project to track the appropriate sequence
+of linking for the final application whether CPU-only or CUDA enabled::
+
+  add_library(myconsumer SHARED ...)
```

Comment:
Note that the same description given above for ``celeritas_target_link_libraries`` is also true for ``celeritas_add_library``
```suggestion
  celeritas_add_library(myconsumer SHARED ...)
```
Furthermore if the library contains CUDA code, using celeritas_add_library is compulsory (I forgot if you can mix RDC library with non RDC library which might be the only case were you might not need it).

Reply:
Good call. on this, I've updated this in all of the locations touched here.